recipes/m2crypto/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: m2crypto
+  version: !!str 0.23.0
+
+source:
+  fn: M2Crypto-0.23.0.tar.gz
+  url: https://pypi.python.org/packages/source/M/M2Crypto/M2Crypto-0.23.0.tar.gz
+  md5: 89557730e245294a6cab06de8ad4fb42
+
+  patches:
+    - setup.patch
```

Comment:
Could you please explain the purpose of this patch? Also, has this been sent upstream (e.g. a PR opened)?